testsuite/kuadrant/__init__.py

```diff
@@ -28,6 +28,10 @@ def deployment(self):
         with self.context:
             return selector("deployment", labels={"app": self.spec_name}).object(cls=Deployment)
 
+    def name(self):
+        """Overrides `name` method from `apiobject` so it returns name of Kuadrant section"""
+        return self.spec_name
```

Comment:
I wouldn't override apiobject properties. I'd use/rename spec_name if there is no other override reason

Reply:
The idea behind this is that both `kuadrant.limitador` and `limitador` standalone (I know that we don't have it yet). Have the same interface and can be interchangeable in tests (same as authorino). So if I call `authorino.name()` I want to get the name of resource, or the name of subcomponent of kuadrant. Does this make a sense?